doc/pages/command-parsing.asciidoc

```diff
@@ -42,16 +40,11 @@ No other escaping takes place in quoted strings.
 
 == Balanced Strings
 
-If a word starts with `%X` with `X` a nestable punctuation character (one
-of `(`, `[`, `{` and `<`), it is parsed as a balanced string whose closing
-delimiter is the matching character of its opening delimiter (respectively
-`)`, `]`, `}` and `>`).
-
-A balanced string contains every character (including whitespaces) until
-a closing delimiter is found, and opening and closing delimiters are
-balanced inside the string (each opening delimiter appearing inside the
-string have been closed by a matching closing delimiter).
+If a word starts with `%X` with `X` a nestable punctuation character (one of `(`, `[`, `{` and `<`),
+it is parsed as a balanced string
+whose closing delimiter matches that of its opening delimiter (respectively `)`, `]`, `}` and `>`).
 
+A balanced string contains every character (including whitespaces).
```

Comment:
this sentence doesn't make much sense by itself

Reply:
In the previous paragraph, we're told what a balanced string is. So I don't know what other information there is to take from the original passage:

> A balanced string contains every character (including whitespaces) until a closing delimiter is found, and opening and closing delimiters are balanced inside the string (each opening delimiter appearing inside the string have been closed by a matching closing delimiter).

- The "until a closing delimiter is found" is redundant.
- The "opening and closing delimiters are balanced inside the string" is also redundant
- Lastly, the parenthesis is also redundant.

The only new information is that about whitespace. This mirrors the section on **Quoted Strings**. Except there, we're also told about doubling the character to escape it, which I think is a separate idea and should be in a separate paragraph.

Reply:
Since there is not really any new information, we could drop this sentence. Haven't tried that though.

Reply:
I removed it.